test/range_access.cpp

```diff
@@ -11,13 +11,16 @@
 // Project home: https://github.com/caseycarter/cmcstl2
 //
 #include <stl2/iterator.hpp>
+#include <stl2/detail/algorithm/find.hpp>
 #include <iostream>
 #include <vector>
 #include "simple_test.hpp"
 
+namespace ranges = __stl2;
```

Comment:
There are a ton of non-functional changes in this file replacing uses of `__stl2` with `ranges` and indenting namespace contents. The actual function change is the addition of `test_string_view_p0970` on line 172.